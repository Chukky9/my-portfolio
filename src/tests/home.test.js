import { screen, render } from '@testing-library/react';
import Home from '../components/home';

test('renders home page successfully', () => {
    const { container } = render(<Home/>)
    const heading = screen.getByRole('heading', { level: 1 })

    expect(heading).toBeInTheDocument()
    expect(heading).toHaveTextContent('Hello, I am Chukwuemeka')
    expect(container).toMatchSnapshot()
})