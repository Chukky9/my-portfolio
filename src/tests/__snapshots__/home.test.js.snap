// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders home page successfully 1`] = `
<div>
  <section
    class="welcome-section"
  >
    <div
      class="laptop-div"
    >
      <i
        aria-hidden="true"
        class="fa fa-laptop"
      />
    </div>
    <h1>
      Hello, I am Chukwuemeka
    </h1>
    <p>
      "I help design and build functional and user-friendly websites for businesses and brands"
    </p>
  </section>
</div>
`;
